virttest/gluster.py

```diff
@@ -422,3 +456,55 @@ def gluster_nfs_disable(vol_name, session=None):
         output = decode_to_text(process.system_output(cmd2))
 
     return 'nfs.disable: on' in output
+
+
+def setup_or_cleanup_gluster(is_setup, vol_name, brick_path="", pool_name="",
+                             file_path="/etc/glusterfs/glusterd.vol", **kwargs):
+    """
+    Set up or clean up glusterfs environment on localhost
```

Comment:
Not only on localhost, so please refine the description.

Reply:
updated